mantle/cmd/kola/testiso.go

```diff
@@ -979,9 +979,9 @@ func testAsDisk(ctx context.Context, outdir string) (time.Duration, error) {
 	return awaitCompletion(ctx, mach, outdir, completionChannel, nil, []string{liveOKSignal})
 }
 
-// iscsi_butane_setup.yaml contain the full butane config but here is an overview of the setup
+// iscsi_butane_setup.yaml contains the full butane config but here is an overview of the setup
 // 1 - Boot a live ISO with two extra 10G disks with labels "target" and "var"
-//   - Format and mount `virtio-var` to var
+//   - Format and mount `virtio-var` to /var
 //
 // 2 - target.container -> start an iscsi target, using quay.io/jbtrystram/targetcli
```

Comment:
This can be updated to quay.io/coreos-assembler/targetd

Reply:
Will update